src/uk/gov/defra/ffc/DefraUtils.groovy

```diff
@@ -131,6 +131,67 @@ def provisionInfrastructure(target, item, parameters) {
   }
 }
 
+def generatePrNames(dbName, prCode) {
+  def prSchema = "pr$prCode"
+  def prUser = "${dbName}_$prSchema"
+  return [prSchema, prUser]
+}
+
+def runPsqlCommand(dbHost, dbUser, dbName, sqlCmd) {
+  sh returnStdout: true, script: "psql --host=$dbHost --username=$dbUser --dbname=$dbName --no-password --command=\"$sqlCmd;\""
+}
+
+def provisionPrRoleAndSchema(host, dbName, jenkinsUserCredId, prUserCredId, prCode, useIfNotExists) {
```

Comment:
Should this include 'Db' as well, e.g. `provisionPrDbRoleAndSchema` / `destroyPrDbRoleAndSchema`? I'm thinking we'll modify the `provisionInfrastructure` and `destroyInfrastructure`  to be `provisionSqsQueue` and `destroySqsQueue`, although in light of this naming convention it should probably be `provisionPrSqsQueue` / `destroyPrSqsQueue`

Reply:
Agreed, will add database to the func names